treeAnalysis/caloheader.h

```diff
@@ -56,7 +56,6 @@ void setBOOLClusterArrayToZero(bool* &arrayinput){
 }
 // conversion functions for processing
 float* EtaPhiFromIndices(int ieta,int iphi,float energy = 0, int caloSelect = 0);
-TVector3 TowerPositionVectorFromIndices(int ieta,int iphi, int caloSelect = 0);
```

Comment:
Why remove the forward declaration here?

Reply:
I don't like to have, but for whatever reason when I added a call to this function [in treeProcessing.C](https://github.com/tlprotzman/AnalysisSoftwareEIC/blob/bdacb5c536b522322334cbb2396bba1c90fcdd29/treeAnalysis/treeProcessing.C#L608) ROOT wasn't running with the declaration, saying it couldn't distinguish between the two call signatures despite it just being a forward declaration.  If you'd like I can investigate it more, honestly I had forgotten about it.

Reply:
Okay, that's odd, but fair enough - no need to look into it more right now